### ISSUE TITLE
PMID entry missing from Fetch text mining v. 0.4.4

### ISSUE THREAD

**Issue by LCCarmody** [pnrobinson/hphenote#46]
@ielis 

HI, I was testing the v,0.4.4 and used the text mining tool.  The fetching of terms worked(!), but there is an issue.  There is no where to add the PMID ID on the text mining tool.  I am aware of the "Source" data entry spot on the main page, but previously, I had only used that when adding a single HPO term.  

So, it allows me to use the 'Fetch text mining' tool.  It allows me to add text, analyze, and add selected terms.  If you hit 'Confirm & close", you get the error, "Warning-attempting to update annotation without valid PMID".  However, at that point, there is no way to add a PMID and it deletes all of your work.  This is not ideal.  

Could you either restrict opening the fetch text mining tool until after you have added a 'source' or have another way to add a source after the fact?  Much data will be lost otherwise.

**Comment by kingmanzhang** [comment]
I suggest we modify from the PhenoteFX side. Probably we do not mandate the presence of source?

**Comment by LCCarmody** [comment]
@kingmanzhang I think so, too.

related: I found another issue.  I used the fetch mining tool...then went back to the single phenotype addition...and then went back to the 'fetch text mining' tool.  Before I selected the 'fetch mining tool' a second time, I clicked the box for 'use last source?' button and then went to 'fetch mining tool'.  It did not register the source and I lost all the work I did in the 'fetch text mining' tool.
